pode/geometry_utils.py

```diff
@@ -21,12 +21,12 @@
 from shapely.affinity import (rotate,
                               scale,
                               translate)
-from shapely.geometry import (LineString,
+from shapely.geometry import (GeometryCollection,
+                              LineString,
                               LinearRing,
-                              MultiPoint,
                               Point,
                               Polygon,
-                              asMultiPoint)
+                              MultiPolygon)

```

Comment:
were sorted alphabetically and now they are not, or it was just a coincidence?

Reply:
I forgot to tell Pycharm to sort them for me. I'm going to fix it now.